src/domains/ecommerce-app/api/ecommerce-helpdesk-api/v1/_openapi.json.tpl

```diff
@@ -999,6 +1109,302 @@
             ]
           }
         }
+      },
+      "SearchPaymentMethodRequestFiscalCode": {
+        "type": "object",
+        "description": "Search transaction by user fiscal code",
+        "properties": {
+          "type": {
+            "type": "string"
+          },
+          "userFiscalCode": {
+            "type": "string",
+            "minLength": 16,
+            "maxLength": 16
+          }
+        },
+        "required": [
+          "type",
+          "userFiscalCode"
+        ],
+        "example": {
+          "type": "USER_FISCAL_CODE",
+          "userFiscalCode": "MRGHRN97L02C469W"
+        }
+      },
+      "SearchPaymentMethodRequestEmail": {
+        "type": "object",
+        "description": "Search transaction by user fiscal code",
+        "properties": {
+          "type": {
+            "type": "string"
+          },
+          "userEmail": {
+            "type": "string",
+            "pattern": "(?:[a-zA-Z0-9!#$%&'*+\\/=?^_`{|}~-]+(?:\\.[a-zA-Z0-9!#$%&'*+\\/=?^_`{|}~-]+)*|\"(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21\\x23-\\x5b\\x5d-\\x7f]|\\\\[\\x01-\\x09\\x0b\\x0c\\x0e-\\x7f])*\")@(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?\\.)+[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?|\\[(?:(?:25[0-5]|2[0-4]\\d|[01]?\\d\\d?)\\.){3}(?:25[0-5]|2[0-4]\\d|[01]?\\d\\d?|[a-zA-Z0-9-]*[a-zA-Z0-9]:(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21-\\x5a\\x53-\\x7f]|\\\\[\\x01-\\x09\\x0b\\x0c\\x0e-\\x7f])+)\\])"
+          }
+        },
+        "required": [
+          "type",
+          "userEmail"
+        ],
+        "example": {
+          "type": "USER_EMAIL",
+          "userEmail": "mario.rossi@pagopa.it"
+        }
+      },
+      "PaypalDetailInfo": {
+        "type": "object",
+        "description": "PaypalDetailInfo",
+        "properties": {
+          "type": {
+            "type": "string",
+            "description": "property discriminator, used to discriminate the wallet detail. Fixed value 'PAYPAL'",
+            "example": "PAYPAL"
+          },
+          "creationDate": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "ppayEmail": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "type",
+          "creationDate",
+          "ppayEmail"
+        ]
+      },
+      "BankAccountDetailInfo": {
+        "type": "object",
+        "description": "BankAccountDetailInfo",
+        "properties": {
+          "type": {
+            "type": "string",
+            "description": "property discriminator, used to discriminate the wallet detail. Fixed value 'BANK_ACCOUNT'",
+            "example": "BANK_ACCOUNT"
+          },
+          "creationDate": {
+            "type": "string",
```

Comment:
Add a brief description for all those response fields

Reply:
Fixed with a81a3bf76ab2eae4a3042b4369aaecf5458f5233